backend/core/detection.py

```diff
@@ -619,12 +651,32 @@ def detect_dplane_mitm_hijack(
             # Placeholder for mitm detection  (not supported)
             return "-"
 
+        @exception_handler(log)
+        def detect_leak_hijack(
+            self, monitor_event: Dict, prefix_node: radix.Radix, *args, **kwargs
+        ) -> Tuple[int, str]:
+            """
+            Route leak hijack detection
+            """
+            for item in prefix_node.data["confs"]:
+                if "no-export" in item["policies"]:
```

Comment:
here we need to add a monitoring AS check (don't know if we will have them hardcoded or no)

Reply:
please create a separate issue for this. It is too much for the current issue (finding valid monitors requires much more work).

Reply:
or could we simply add all monitors to a redis set? What do you think?

Reply:
this set will simply get larger, no need to remove ASNs or anything

Reply:
yes we can have a redis set with the monitoring ASes 👍

Reply:
https://github.com/FORTH-ICS-INSPIRE/artemis/issues/102